twitter/twitter.go

```diff
@@ -0,0 +1,43 @@
+package twitter
+
+import (
+	"fmt"
+
+	"github.com/dghubble/go-twitter/twitter"
+	"github.com/dghubble/oauth1"
+)
+
+// Client ...
+type Client struct {
+	ConsumerKey    string
+	ConsumerSecret string
+	AccessToken    string
+	AccessSecret   string
+	Client         *twitter.Client
+}
+
+// NewClient ...
+func NewClient(consumerKey, consumerSecret, accessToken, accessSecret string) *Client {
```

Comment:
As the method is very small, no need for this big names. In go it is common to favor short variable names (as long it does not make readability worse).